identity-apps-core/apps/authentication-portal/src/main/webapp/includes/language-switcher.jsp

```diff
@@ -78,26 +78,30 @@
      * @param name - Name of the cookie
      * @param value - Value to be stored
      * @param days - Expiry days
+     * @param options - Additional options for the cookie such as `httpOnly` and `secure`.
      */
-    function setCookie(name, value, days) {
+    function setCookie(name, value, days, options) {
         var expires = "";
         var domain = ";domain=" + URLUtils.getDomain(window.location.href);
 
         if (days) {
-            const date = new Date();
-            date.setTime(date.getTime() + (days*24*60*60*1000));
+            var date = new Date();
```

Comment:
Do we still need to avoid `const` since es6 is widely supported?

https://caniuse.com/?search=es6

Reply:
Better to align with ES5 for now since that's the consistent ECMA standard we've been following.
Mixture of the two is never good.

The proper thing to do is we need to move these inline script blocks to dedicated files and then use a compiler.
This will take time.